playbooks/roles/sslh/templates/sslh.j2

```diff
@@ -14,4 +14,4 @@ RUN=yes
 # binary to use: forked (sslh) or single-thread (sslh-select) version
 DAEMON=/usr/sbin/sslh
 
-DAEMON_OPTS="-u sslh -p {{ ansible_default_ipv4.address }}:443 --ssh 127.0.0.1:{{ ssh_port }} --openvpn 127.0.0.1:{{ openvpn_port }} --ssl 127.0.0.1:{{ nginx_port }} -P /var/run/sslh/sslh.pid"
+DAEMON_OPTS="-u sslh -p {{ ansible_default_ipv4.address }}:443 --ssh 127.0.0.1:{{ ssh_port }} {% if streisand_openvpn_enabled %} --openvpn 127.0.0.1:{{ openvpn_port }} {% endif %} --ssl 127.0.0.1:{{ nginx_port }} -P /var/run/sslh/sslh.pid"
```

Comment:
I think this is fine for now but it would be a lot less messy to read if we used the SSLH config file instead of the systemd defaults and the `DAEMON_OPTS`.

I opened https://github.com/jlund/streisand/issues/827 to remember that we can clean this up a little bit in the future. I don't see an easy way to avoid having `{% if  %}` blocks based on module vars but I think having it as a config file instead of a set of CLI parameters will help.